js/logo.js

```diff
@@ -4852,17 +4848,32 @@ function Logo(pitchtimematrix, pitchdrummatrix, rhythmruler,
         } else if (logo.blocks.blockList[blk].isArgBlock() || logo.blocks.blockList[blk].isArgClamp()) {
             switch (logo.blocks.blockList[blk].name) {
             case 'loudness':
-                try {  // DEBUGGING P5 MIC
-                    if (!logo.mic.enabled) {
-                        logo.mic.start();
+                try {  
+                    var analyser = new Tone.Analyser({
+                        "type" : "waveform",
+                        "size" : 256
```

Comment:
I think this is the sample size so it must match the 1024 below. Maybe define a constant?